### PR TITLE
Added missing dep is-npm and mocha

### LINKED ISSUE #3: Cannot find module 'is-npm'
Cypress-parallel fails to start if is-npm is not installed in the project
Installing is-npm manually fixes the issue

### PR DESCRIPTION
**Pull request by mribichich**
/closes #3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "homepage": "https://github.com/tnicola/cypress-parallel#readme",
     "dependencies": {
         "cypress": "^4.6.0",
-        "cypress-parallel": "file:lib"
+        "cypress-parallel": "file:lib",
+        "is-npm": "^5.0.0"
```
Comment (tnicola) [inline package.json:24]: This dependency should be added to `lib/package.json`.

Comment (mribichich) [reply package.json:24]: sorry didn't see that you had a separate package.json

**Comment by tnicola** [comment]
@mribichich Thanks. This should also fix #3

**Comment by Yethal** [comment]
Thanks

**Pull request merged**